src/node_quic_util.h

```diff
@@ -36,25 +37,75 @@ constexpr uint16_t NGTCP2_APP_NOERROR = 0xff00;
 
 constexpr size_t MIN_INITIAL_QUIC_PKT_SIZE = 1200;
 constexpr size_t NGTCP2_SV_SCIDLEN = 18;
+constexpr size_t TOKEN_RAND_DATALEN = 16;
+constexpr size_t TOKEN_SECRETLEN = 16;
 constexpr size_t DEFAULT_MAX_STREAM_DATA_BIDI_LOCAL = 256_k;
 
 class SocketAddress {
  public:
+  static size_t GetMaxPktLen(const sockaddr* addr) {
+    return addr->sa_family ? NGTCP2_MAX_PKTLEN_IPV6 : NGTCP2_MAX_PKTLEN_IPV4;
+  }
+
+  static int ToSockAddr(
```

Comment:
This function looks quite similar to the [sockaddr_for_family](https://github.com/nodejs/node/blob/230f1f2aa4aa2d5800214c4894d1c13a3f153dd0/src/udp_wrap.cc#L177). Perhaps as some later stage we should have an util class that both can use.

Reply:
let's consolidate later